it/src/test/java/org/corfudb/universe/scenario/ConcurrentClusterResizeIT.java

```diff
@@ -102,7 +105,7 @@ public void concurrentClusterResizeTest() {
             ));
 
             // Wait for layout servers to change
-            waitForLayoutServersChange(size -> size == numNodes, corfuClient);
+            waitForLayoutChange(layout -> layout.getSegments().size() == 1, corfuClient);
```

Comment:
please also save the old check, that layout size equals to `numNodes`, it is a more important check

Reply:
+1